.github/workflows/build-images-branches.yaml

```diff
@@ -13,3 +13,4 @@ jobs:
     with:
       kuadrantOperatorVersion: ${{ github.ref_name }}
       kuadrantOperatorTag: ${{ github.ref_name }}
+      quayImageExpiry: 1w
```

Comment:
Abitary value of when to expire branch images. If anyone has a better suited value, I can update it to this value

Reply:
This will also affect releases branches currently which is probably something we don't want 🤔